OpenSourcePolitics/decidim-app
# Fix invariable images

#### :tophat: Description
Rescue when image is not variable

#### :pushpin: Related Issues
- [Notion card](https://www.notion.so/opensourcepolitics/EDP-ActiveStorage-InvalidError-55a76c43aa3b4976847bd8944e486569?pvs=4)

#### Testing
Example:
* Log in as user
* Create a proposal with attachment
* Upload webp file
* See image on preview

#### Tasks
